packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsList.tsx

```diff
@@ -46,27 +46,38 @@ const Accounts = ({
     const { params } = selectedAccount;
     const isSkeletonShown = discoveryInProgress || (type === 'coinjoin' && coinjoinIsPreloading);
 
+    const router = useSelector(state => state.router);
```

Comment:
how about just adding one more condition into `isSelected` `router.params &&` 

if `router.params` are available, then use this logic, if they are not there. Do not select any account at all